relayer/src/channel.rs

```diff
@@ -420,54 +426,77 @@ impl Channel {
         let a_channel = self
             .src_chain()
             .query_channel(&self.src_port_id(), src_channel_id, Height::zero())
-            .map_err(|_| ChannelError::Failed("Failed to query source chain".into()))?;
+            .map_err(|_| {
+                ChannelError::Failed(format!(
+                    "failed to query source chain {}",
+                    self.src_chain().id()
+                ))
+            })?;
 
         let b_channel = self
             .dst_chain()
             .query_channel(&self.dst_port_id(), dst_channel_id, Height::zero())
-            .map_err(|_| ChannelError::Failed("Failed to query destination chain".into()))?;
+            .map_err(|_| {
+                ChannelError::Failed(format!(
+                    "failed to query destination chain {}",
+                    self.dst_chain().id()
+                ))
+            })?;
 
         match (a_channel.state(), b_channel.state()) {
             (State::Init, State::TryOpen) | (State::TryOpen, State::TryOpen) => {
                 let event = self.flipped().build_chan_open_ack_and_send().map_err(|e| {
-                    error!("Failed ChanAck {:?}: {}", self.a_side, e);
+                    error!("failed ChanAck {:?}: {}", self.a_side, e);
                     e
                 })?;
 
                 info!("done {} => {:#?}\n", self.src_chain().id(), event);
```

Comment:
Since this is failing, the log info saying this is done is a bit confusing.

Reply:
14112f07